File: ManageBE/Manage/Models/FileDocumenti.cs
namespace Manage.Models
{
    public class FileDocumenti
    {
        public int Id { get; set; }
        public string NomeFile { get; set; }
        public string EstensioneFile { get; set; }
        public string PercorsoFile { get; set; }
        public DateTime DataInserimentoFile { get; set; } = DateTime.Now;

        // Foreign Key per la relazione con Documenti
        public int DocumentiId { get; set; }
        public Documenti Documento { get; set; }
    }
}

File: ManageFE/src/app/M-Models/FileDocumenti.ts
export interface FileDocumentiData {
    id: number;
    nomeFile: string;
    estensioneFile: string;
    percorsoFile: string;
    dataInserimentoFile: Date;
    documentiId: number;
    isTemporary?: boolean;
}

export interface FileDocumentiDataValue {
    id: number;
    nomeFile: string;
    estensioneFile: string;
    dataInserimentoFile: Date;
    documentiId: number;
}